usaspending_api/etl/tests/integration/test_load_to_delta.py

```diff
@@ -60,6 +277,20 @@ def equal_datasets(psql_data: List[Dict[str, Any]], spark_data: List[Dict[str, A
                 psql_val_utc = psql_val.astimezone(utc_tz)
                 psql_val = psql_val_utc.replace(tzinfo=None)
 
+            # In the case of a list make sure they are ordered prior to comparison
+            if isinstance(psql_val, list):
+                psql_val = sorted(psql_val)
+
+            if isinstance(spark_val, list):
+                spark_val = sorted(spark_val)
+
+            # Handle JSON formatted strings where the only difference is spacing between keys
+            if isinstance(psql_val, str):
+                psql_val = psql_val.replace(" ", "")
+
+            if isinstance(spark_val, str):
+                spark_val = spark_val.replace(" ", "")
```

Comment:
isn't this removing all spaces in ALL string fields? How does this ensure it's just for JSON strings?
And what about spaces within the value of JSON fields?

Reply:
It does, but given it is just a test case I was trying to think of the best way to verify data matches without making it too complicated. Can take another pass thinking about it.

Reply:
I may have missed something in the way github redacts code... if this is just a function for _your_ test case, then it's all good. But if this is the shared function used by many test cases, then all the string fields are going to get transformed for every table's test case that uses that function.